<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MedicineSearchFunc</class>
 <widget class="QWidget" name="MedicineSearchFunc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="SearchDrug">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>SearchDrug</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GetDetail">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>GetDrugDetail</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="DrugBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="DetailBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="DrugName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="DrugId">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
